version/version.go

```diff
@@ -0,0 +1,47 @@
+// Package version records the verion of the ts-bridge project.
+// It also includes helper methods to return a user-agent that can be used in requests.
+package version
+
+import (
+	"fmt"
+	"runtime"
+
+	"google.golang.org/api/googleapi"
+)
+
+const (
+	// Project is the name of this project.
+	Project = "ts-bridge"
+
+	// Ver is the current version of this project.
+	Ver = "0.1"
+)
+
+// Variables that can be changed for tests.
+var (
+	// Revision is the main version revision of the Project.
+	version = Revision()
+
+	// rt is the Go runtime version.
+	rt = runtime.Version()
+
+	// goos is the os.
+	goos = runtime.GOOS
+
+	// goarch is the architechture.
+	goarch = runtime.GOARCH
+
+	// googClient is the google API client version.
+	googClient = googleapi.UserAgent
+)
+
+// Revision returns the current revision of this project.
+func Revision() string {
```

Comment:
I know this is slightly out of scope, but can I ask you to pipe it to `main.go` so it prints the revision if `--version` flag is passed?

We will need it for CI/releasing.

Feel free to push back if you want to make this a separate task.

Reply:
Done.

Reply:
Not quite. Sorry, my bad. Basically we need the standard UNIX behaviour here. If `--version` flag is passed - print the version and exit immediately with 0.

E.g.:
```
λ fly --version
5.6.0
λ echo $?
0
```
This is needed for packaging automation so the version in the app matches everything else.

Reply:
Done.